swarm-private/templates/bootnode.statefulset.yaml

```diff
@@ -39,15 +39,33 @@ spec:
         - sh
         - -ac
         - >
-          /bootnode
+          /run.sh
+          --debug
+          --bootnode-mode
           --nodekey=/keys/bootnode.key
-          --addr=:30301
           --verbosity={{ .Values.bootnode.config.verbosity }}
+          --maxpeers 2000
+          --nat=ip:$POD_IP
+          --port=30301
+          --bzznetworkid={{ .Values.bootnode.config.bzznetworkid }}
+          --bootnodes=enode://0db3ed33199e4168944c2c0a2b5994c8521ccb942f43fb432f481d88a02c6dee71d4e1da3727a29d4952d7653c862f74c6822d68abf6ae63d9251bbece8b87a5@10.0.55.155:30499
```

Comment:
This doesn't look right, having the node address and IP hardcoded in here.

Reply:
Yes, this needs to be fixed - we need a way to set `empty` bootnodes. If we don't specify any `bootnodes`, then the code currently picks the default bootnodes (which we don't want).

So yeah - we have to fix `swarm` code to be able to have no bootnodes (we don't want bootnodes to connect to anything basically)... or maybe we will want bootnodes to connect to each other.

Reply:
I see. In that case, could you put a comment next to that line explaining that the flag is only needed as a workaround because currently it won't work without it.

Reply:
empty `--bootnodes` list is not accepted right now. If https://github.com/ethereum/go-ethereum/pull/18509 is approved, then we can pass an empty list. Otherwise I will think of another solution.